app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'AuthorId'
```

Comment:
- [ ] I like the association here between a user's table and comments table, but wait a bit, we are trying to create a custom table name attribute right? We want a user to be called the author (comment create). We can directly say that comment belongs to the author. 

Hint: `belongs_to :author, class_name: 'User'`